pyproject.toml

```
@@ -0,0 +1,46 @@
+[tool.poetry]
+name = "gurkbot"
+version = "0.1.0"
+description = "Our community bot, used for running the server."
+authors = ["Your Name <you@example.com>"]
```

lol, that should be changed 
```suggestion
authors = ["The Gurkult Community <noreply@gurkult.com>"]
```

---

Awesome, will do that

---

Done